common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -75,6 +75,7 @@
         if ((unsafeUnavailabilityCause = EXPLICIT_NO_UNSAFE_CAUSE) != null) {
             direct = null;
             addressField = null;
+            unsafeUnavailabilityCause = new UnsupportedOperationException("Unsafe explicitly disabled");
```

Comment:
Why is this added ? It is not from our fork neither in the `4.1.48` tag of netty. I think this can be removed.

Reply:
Thanks, this was a leftover from our old branch, removed https://github.com/sirensolutions/netty/pull/17/commits/2d1313a5f976fb93597a34fe12675797f1a13089